juniper_official/chassis/check-psm-power-usage-state.rule

```diff
@@ -91,41 +98,58 @@ healthbot {
                 description "Sets health based on increasing psm power usage";
                 frequency 1offset;
                 /*
-                 * Sets color to red when psm-power-usage is greater than psm-power-usage-threshold
+                 * Sets color to green when psm-power-usage is lesser than psm-power-usage-low-threshold
                  * 
                  */				
-                term is-power-usage-abnormal {
+                term power-usage-normal {
                     when {
-                        greater-than-or-equal-to "$psm-power-usage" "$psm-power-usage-threshold";
+                        less-than "$psm-power-usage" "$psm-power-usage-low-threshold";
+                        does-not-match-with "$psm-state" offline {
```

Comment:
Wouldn't it be better to compare with what we we are expecting?

Reply:
Changed to Online the new MR.